pkg/engine/engine.go

```diff
@@ -108,7 +115,8 @@ func (e *engine) Mutate(
 		policyResponse, patchedResource := e.mutate(ctx, logger, policyContext)
 		response = response.
 			WithPolicyResponse(policyResponse).
-			WithPatchedResource(patchedResource)
+			WithPatchedResource(patchedResource).
+			WithPolicyResponse(policyResponse)
```

Comment:
Isn't this a duplicate of line [117](https://github.com/kyverno/kyverno/pull/11453/files#diff-9a4b0f64963e54937214c78c91c3d404eaef74b571e89f370ed7de3c9ff86ca1R117)?

Reply:
Yeah, likely from a bad rebase. will remove